automations/js/src/project_automation/prs.mjs

```diff
@@ -84,7 +84,7 @@ export const main = async (octokit, core) => {
   if (eventName === 'pull_request_review') {
     if (pr.isDraft) {
       await prBoard.moveCard(prCard.id, prBoard.columns.Draft)
-    } else {
+    } else if (!pr.isMerged) {
```

Comment:
Just adding an explanatory comment here, otherwise it's potentially a little obscure.

```suggestion
    } else if (!pr.isMerged) {
      // Don't touch merged PRs to avoid race condition when a PR is merged right after the
      // review and is alredy in a merged state when re-retrieved by the workflow, even though the
      // triggering event was the second review. In that case we want our handling of merged PRs to
      // take precedence, rather than updating the status based on the second review.
```

Reply:
Added in 752ef76. Couldn't commit this because of a typo ("alredy") and because I thought line-length might break linting.